test/Kestrel.FunctionalTests/AddressRegistrationTests.cs

```diff
@@ -484,14 +470,16 @@ public void ThrowsWhenBindingToIPv4AddressInUse()
         [IPv6SupportedCondition]
         public void ThrowsWhenBindingToIPv6AddressInUse()
         {
+            TestApplicationErrorLogger.ThrowOnCriticalErrors = false;
```

Comment:
?

Reply:
We inject a TestApplicationErrorLogger by default and it wraps invalid operation exceptions with an aggregate exception. These tests require the invalid operation exceptions so I'm disabling the throwing behaviour.

Reply:
This test is expecting an error log, and the test logger throws by default if you log an error.